model/cloudevents/spans.yaml

```diff
@@ -1,6 +1,6 @@
 groups:
   - id: cloudevents
-    type: span
+    type: attribute_group
```

Comment:
@joaopgrassi it seems we're saying that cloudevents are just attributes on messaging semconv, do you think we can change the group type?

Reply:
Yes, I think that is fine and should be like that anyway!